custom-tsdb/values-primary.yaml

```diff
@@ -4,6 +4,12 @@ kubecostModel:
   promClusterIDLabel: YOUR_PROMETHEUS_CLUSTER_KEY # set this to the same as the key used for prometheus cluster labels
 kubecostProductConfigs:
   clusterName: YOUR_PROMETHEUS_CLUSTER_VALUE # set this to the name of the primary cluster. ideally the same as the value that promClusterIDLabel would be
+prometheus:
+  server:
+    global:
+      external_labels:
+        # cluster_id should be unique for all clusters and the same value as .kubecostProductConfigs.clusterName
+        cluster_id: kubecostProductConfigs_clusterName
```

Comment:
To avoid confusion, I think we should use the same placeholder as kubecostProductConfigs.clusterName . Example:

cluster_id: YOUR_PROMETHEUS_CLUSTER_VALUE #Value must match with kubecostProductConfigs.clusterName

Reply:
good call out!